packages/experimental/RadioGroup/src/Radio/Radio.styling.ts

```diff
@@ -19,21 +19,24 @@ export const stylingSettings: UseStylingOptions<RadioProps, RadioSlotProps, Radi
           minHeight: 20,
           marginTop: 0,
           marginRight: globalTokens.spacing.s,
+          borderColor: tokens.borderColor,
+          borderWidth: tokens.borderWidth,
+          borderStyle: tokens.borderStyle,
           borderRadius: tokens.borderRadius,
```

Comment:
You might want to use `...borderStyles.from(tokens, theme)` for this.

Reply:
Replaced, thanks!